torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -381,13 +381,47 @@ def get_meta_value(argument: torch.fx.node.Argument):
         ):
             return False
 
-        if any(
-            get_meta_value(n.args[0]).dim() != get_meta_value(n.args[1]).dim()
-            or not all(
-                get_meta_value(n.args[0]).size(i) == get_meta_value(n.args[1]).size(i)
-                or get_meta_value(match.kwargs["other"]).size(i) == 1
-                for i in range(get_meta_value(n.args[0]).dim())
+        def _check_input_sizes(n, computation_op):
+            computation_node = (
+                n.args[0] if n.args[1] is match.kwargs["other"] else n.args[1]
+            )
+            assert computation_node.target == computation_op
+            computation_node_size = get_meta_value(computation_node).size()
+            if computation_op is mkldnn._linear_pointwise.default:
+                if len(computation_node_size) >= 2:
+                    broadcast_sizes = [
+                        torch.Size(
+                            [1 for _ in range(len(computation_node_size) - 1)]
+                            + [computation_node_size[-1]]
+                        ),
+                        torch.Size([1 for _ in range(len(computation_node_size))]),
+                    ]
+                else:
+                    broadcast_sizes = [
+                        torch.Size([1 for _ in range(len(computation_node_size))]),
+                    ]
+            else:
+                broadcast_sizes = [
```

Comment:
`assert len(computation_node_size) >= 2` here?

Reply:
Added, thanks for your comments!